sys/include/net/conn.h

```diff
@@ -15,6 +15,7 @@
  * About
  * =====
  *
+ * @code
```

Comment:
You're right it does, just not on the outdated branch I looked at it on (weird sentence o.o). nevermind me then! (fences are installed in conn now, too)